src/web/src/components/Posts/Post.tsx

```diff
@@ -302,6 +308,14 @@ const PostComponent = ({ postUrl }: Props) => {
           dangerouslySetInnerHTML={{ __html: post.html }}
         />
       </div>
+      <div className={classes.contentFooter}>
+        <p>
+          Written by:{' '}
```

Comment:
I'm not sure how I feel about `Written by`, we already show the user on the right, why add it again at the bottom? I do think the line is nice though.

What do others think?